Description: Disable annotation without ghci
 Can be removed if ghc ignores annotation without an interpreter:
 http://hackage.haskell.org/trac/ghc/ticket/4268
Author: Joachim Breitner <nomeata@debian.org>
Bug-Debian: http://bugs.debian.org/640088
Forwarded: no

Index: haskell-vector-0.9.1/Data/Vector/Fusion/Stream/Monadic.hs
===================================================================
--- haskell-vector-0.9.1.orig/Data/Vector/Fusion/Stream/Monadic.hs	2012-02-10 23:48:51.000000000 +0100
+++ haskell-vector-0.9.1/Data/Vector/Fusion/Stream/Monadic.hs	2012-02-10 23:49:04.000000000 +0100
@@ -94,15 +94,19 @@
 import Data.Word ( Word8, Word16, Word32, Word, Word64 )

 #if __GLASGOW_HASKELL__ >= 700
+#ifndef DEBIAN_NO_GHCI
 import GHC.Exts ( SpecConstrAnnotation(..) )
 #endif
+#endif

 #include "vector.h"

 data SPEC = SPEC | SPEC2
 #if __GLASGOW_HASKELL__ >= 700
+#ifndef DEBIAN_NO_GHCI
 {-# ANN type SPEC ForceSpecConstr #-}
 #endif
+#endif

 emptyStream :: String
 {-# NOINLINE emptyStream #-}
